modules/nlu/src/backend/engine.ts

```diff
@@ -158,10 +172,10 @@ export default class Engine implements NLUEngine {
       const intents = await ProcessIntents(
         model.data.input.intents,
         model.languageCode,
-        model.data.artefacts.list_entities,
+        model.data.output.list_entities,
```

Comment:
You're currently in a if branch were `model.data.output` is not defined: this will break,

Please investigate on when this condition occurs, and make sure it does not break.